docs/README.md

```diff
@@ -56,7 +56,9 @@ Note: Any string secret is accessible through SecretSource, but only a secret wi
 
 The plugin allows secrets from Secrets Manager to be used as Jenkins credentials.
  
-A secret will act as one of the following Jenkins [credential types](https://jenkins.io/doc/pipeline/steps/credentials-binding/), based on the `jenkins:credentials:type` tag that you add to it.
+Jenkins must know which [credential type](https://jenkins.io/doc/pipeline/steps/credentials-binding/) a secret is meant to be (e.g. Secret Text, Username With Password), in order to present it as a credential. To do this, **you MUST annotate the secrets with the relevant tags** as shown in the sections below. If the credentials cache is enabled you must also wait for that to refresh before the newly annotated secrets appear in Jenkins.
```

Comment:
Can you mention that this step has to be complete in AWS Secret Manager?
When I first see the readme I seriously thought this is a step within Jenkins.

Reply:
Updated this bit, let me know if it makes sense

Reply:
Thanks.